css/index.css

```diff
@@ -126,6 +261,11 @@ p {
     padding-right: 0;
 }
 
+.bot-content-about {
+    padding: 0 2% 2% 2%;
```

Comment:
Can be replaced with `padding: 0 2% 2%` 

Usually padding and maring have properites for `top right bottom left` but if we ommit the left it will default to the value of right.

For example `padding: a b c` will default to `padding: a b c b`.